kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/algebra/ReporterOps.scala

```diff
@@ -0,0 +1,29 @@
+package com.github.pedrovgs.kuronometer.free.algebra
+
+import cats.free.{Free, Inject}
+import com.github.pedrovgs.kuronometer.KuronometerResults.KuronometerResult
+import com.github.pedrovgs.kuronometer.free.domain.{BuildExecution, Report, SummaryBuildStagesExecution}
+
+sealed trait ReporterOp[A]
+
+final case class ReportBuildExecution(buildExecution: BuildExecution, report: Report) extends ReporterOp[KuronometerResult[BuildExecution]]
+final case class GetTotalBuildExecution() extends ReporterOp[KuronometerResult[SummaryBuildStagesExecution]]
+final case class GetTodayBuildExecution() extends ReporterOp[KuronometerResult[SummaryBuildStagesExecution]]
+
+class ReporterOps[F[_]](implicit I: Inject[ReporterOp, F]) {
```

Comment:
This is an example of the  parametric `F[_]` I mentioned in https://github.com/pedrovgs/Kuronometer/pull/1/files#r116612240